webapp/src/dogma/features/auth/authSlice.ts

```diff
@@ -103,12 +101,10 @@ export const logout = createAsyncThunk('/auth/logout', async (_, { getState, rej
     if (typeof window !== 'undefined') {
       localStorage.removeItem('sessionId');
     }
-  } catch (error) {
-    if (error.response && error.response.data.message) {
-      return rejectWithValue(error.response.data.message);
-    } else {
-      return rejectWithValue(error.message);
-    }
+  } catch (err) {
+    const error: string = ErrorHandler.handle(err);
+    dispatch(createMessageError(error));
+    return rejectWithValue(error);
```

Comment:
Question: Would we create a middleware to display rejected values?
https://redux-toolkit.js.org/rtk-query/usage/error-handling#handling-errors-at-a-macro-level

Reply:
Creating a rtk middleware involves more steps as we can't use ChakraUI `useToast` directly in the middleware.

Reply:
I see. We may create a new action and dispatch the action in a middleware. That said let's consider it later. It is crucial for this PR.